### PR TITLE
Add navigation bar and the community page

### PR DESCRIPTION
**Pull request by oleg-nenashev**
Subj, improves navigation and discoverability

image

Signed-off-by: Oleg Nenashev <o.v.nenashev@gmail.com>

### CODE REVIEW

File: content/home/manifesto.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: "Project Manifesto"
 summary: "What is this project about?"
 date: 2022-02-11
-weight: 2
+weight: 1
 ---
 
 Our goal is to create an open standard for feature flag management to support a robust feature flag ecosystem using cloud native technologies.
```
Comment (AloisReitbauer) [inline content/home/manifesto.md]: Can we make this "The goal of the open feature project ..."  not that people confuse overall project goals with individual goals

File: content/home/participate.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ Sign-up for the news and share your expectations using
 This form collects contact emails and also offers opportunities to share your expectations for the project.
```
Comment (AloisReitbauer) [inline content/home/participate.md:13]: Should we be more specific about what they are interested in? Using, contributing. 

I already got the question what it means to get listed as an interested party.

**Review by AloisReitbauer — CHANGES_REQUESTED** [review]

File: content/home/manifesto.md (hunk 1 above)
Comment (oleg-nenashev) [reply content/home/manifesto.md]:
```suggestion
The goal of the OpenFeature project is to create an open standard for feature flag management to support a robust feature flag ecosystem using cloud native technologies.
```

File: content/home/participate.md (hunk 2 above)
Comment (oleg-nenashev) [reply content/home/participate.md:13]: The form asks about types of contributions, but indeed it worth listing it explicitly

Comment (oleg-nenashev) [reply content/home/participate.md:13]: Should be addressed now

File: config.yml [hunk 3]
```diff
@@ -21,20 +21,23 @@ languages:
...
+              url: https://github.com/openfeatureflags/spec
+              weight: 2
+            - name: Community
+              url: home/community/
+              weight: 4
+            - name: Participate
+              url: home/participate/
+              weight: 4
+            - name: GitHub
+              url: https://github.com/openfeatureflags
```
Comment (beeme1mr) [inline config.yml]:
```suggestion
              url: https://github.com/open-feature
```

File: config.yml [hunk 4]
```diff
@@ -21,20 +21,23 @@ languages:
...
-     #           - name: Categories
-     #             url: categories/
-     #             weight: 10
+        menu:
+            main:
+            - name: Why OpenFeature?
+              url: home/manifesto/
+              weight: 1
+            - name: Specification
+              url: https://github.com/openfeatureflags/spec
```
Comment (beeme1mr) [inline config.yml]:
```suggestion
              url: https://github.com/open-feature/spec
```

File: config.yml [hunk 5]
```diff
@@ -68,11 +71,13 @@ params:
...
         imageWidth: 120
         imageHeight: 120
-      #  buttons:
-      #      - name: Archives
-      #        url: archives
-      #      - name: Tags
-      #        url: tags
+        buttons:
+            - name: GitHub
+              url: https://github.com/openfeatureflags
```
Comment (beeme1mr) [inline config.yml]:
```suggestion
              url: https://github.com/open-feature
```

File: config.yml [hunk 6]
```diff
@@ -68,11 +71,13 @@ params:
...
+            - name: Specification
+              url: https://github.com/openfeatureflags/spec
```
Comment (beeme1mr) [inline config.yml]:
```suggestion
              url: https://github.com/open-feature/spec
```

File: content/home/community.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+
+## Community channels
+
+The community channels and project meetings will be established soon.
+
+## Governance
+
+The project follows the open governance model,
+and invites everyone to participate in the project governance and contribute to the specification.
+Learn more about project governance [here](https://github.com/openfeatureflags/governance).
```
Comment (beeme1mr) [inline content/home/community.md]:
```suggestion
Learn more about project governance [here](https://github.com/open-feature/governance).
```

File: content/home/community.md [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+
+
+## Interested parties
+
+Multiple individual organizations and companies have declared their interested in the project.
+You can find the full list and join [here](https://github.com/openfeatureflags/governance/blob/main/interested-parties.md).
```
Comment (beeme1mr) [inline content/home/community.md]:
```suggestion
You can find the full list and join [here](https://github.com/open-feature/governance/blob/main/interested-parties.md).
```

File: content/home/manifesto.md [hunk 9]
```diff
@@ -2,10 +2,10 @@
 title: "Project Manifesto"
 summary: "What is this project about?"
 date: 2022-02-11
-weight: 2
+weight: 1
 ---
 
-Our goal is to create an open standard for feature flag management to support a robust feature flag ecosystem using cloud native technologies.
+The goal of the OpenFeature project is to create an open standard for feature flag management to support a robust feature flag ecosystem using cloud native technologies.
```
Comment (beeme1mr) [inline content/home/manifesto.md]:
```suggestion
The goal of OpenFeature is to create an open standard for feature flag management to support a robust feature flag ecosystem using cloud native technologies.
```

**Pull request merged**